micronaut-graalpy-bom/build.gradle

```
@@ -0,0 +1,10 @@
+plugins {
+    id 'io.micronaut.build.internal.graal-languages-base'
+    id "io.micronaut.build.internal.bom"
```

Add parenthesis so that it is compatible to both Groovy and Kotlin Gradle DSL.

```suggestion
    id("io.micronaut.build.internal.bom")
```